pkg/command/lint/command.go

```diff
@@ -95,8 +95,7 @@ func Command() *cobra.Command {
 				}
 			}
 			if !atLeastOneObjectFound {
-				fmt.Fprintln(os.Stderr, "Warning: no valid objects found.")
-				return nil
```

Comment:
This breaks backward compatibility, and is more in line with how I'd expect a linter to work -- just emit a warning if there are no valid objects, but don't fail altogether. I would be open to adding a flag, though, that controls this, so that it someone passes it in (say, "--fail-if-no-objects-found"), _then_ we return an error. What do you think?